pantheon-bundle/frontend/src/app/versions.tsx

```diff
@@ -663,7 +663,7 @@ class Versions extends Component<IProps, any> {
                             this.setState({ moduleUrl: url })
                         }
                         this.setState({
-                            usecaseValue: this.state.metadataResults.documentUsecase,
+                            usecaseValue: this.state.metadataResults.documentUseCase,
```

Comment:
In metadataResults, the fieldname is 'documentUsercase', by changing it to upper case 'C', edit metadata no longer works.

Reply:
@xdavidson I think in line 498  is the place where we append the metadata, I don't see anywhere in the backend where we are defining it other than doing the Query. I think I changed all the occurrences on the code from Usecase to UseCase. Not sure if I am missing somewhere else I need to change it.